app/include/lwip/app/ping.h

```diff
@@ -54,6 +54,7 @@ struct ping_option{
 	ping_recv_function recv_function;
 	ping_sent_function sent_function;
 	void* reverse;
+	struct ping_msg *parent_msg;
```

Comment:
I don't believe this is necessary; the `reverse` field right above it should serve admirably for the same purpose?